app/views/my/events/upcoming_events.html.slim

```diff
@@ -10,4 +10,24 @@ ul.nav.nav-tabs
   li
     a = link_to 'Past Events', past_events_my_events_path
 
-h2 Upcoming events
+.container
```

Comment:
bootstrap needs a `row` and `col-**-**` so that your html will be mobile-responsive.
```
.container
  .row
    .col-xs-12
```